packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,124 @@
+# RNTester E2E folder
+
+In this folder we have a the setup for running E2E testing in RNTester via the usage of [Appium](https://appium.io/) and [WebDriverIO](https://webdriver.io/) and [Jest](https://jestjs.io/).
+
+- [Setting up locally](#setting-up-locally)
+  - [(one-off) Setting up Appium](#one-off-setting-up-appium)
+  - [Building RNTester app](#building-rntester-app)
+    - [Building for iOS](#building-for-ios)
+    - [Building for Android](#building-for-android)
+  - [Setting up the RNTester E2E folder](#setting-up-the-rntester-e2e-folder)
+- [Testing the RNTester app E2E](#testing-the-rntester-app-e2e)
+- [Adding new tests (and project structure)](#adding-new-tests-and-project-structure)
+
+## Setting up locally
+
+### (one-off) Setting up Appium
+
+The first step you need to do is to ensure to install the tooling:
+
+```bash
+npm install appium@2.0.0-rc.4 -g
+appium driver install uiautomator2
+appium driver install xcuitest
+```
+
+> More details about drivers in Appium [here](https://appium.github.io/appium/docs/en/2.0/guides/managing-exts/) and [here](https://appium.github.io/appium/docs/en/2.0/quickstart/uiauto2-driver/)
+
+You should not need to run install commands for drivers separately more than once, even if you bump the dep in package.json.
+
+### Building RNTester app
+
+Building manually *.app* and *.apk* is required to run automation tests on local environment.
+
+0. *(optional)* If you previously built RNTester, you may need to clean up build files and Pods:
+
+    ```bash
+    yarn test-e2e-local-clean && yarn install
+    ```
+
+1. Step 1: install packages for the repository, then navigate in the rn-tester folder
+
+    ```bash
+    cd react-native
+    yarn install
+    cd packages/rn-tester
+    ```
+
+Now, depending on the platform, there are some specific steps
```

Comment:
can we have a js script that we can invoke with like:
```bash
yarn run-e2e-test <platform> # <platform> can be "ios" or "android"
```

That automate the steps required to build and run the tests?
This will make it easier also to ask people to contribute as they can just write their tests and run the script.

Reply:
can we have this being a feature to implement as a separate follow up PR? It doesn't sound like a deal breaker and I'd rather get this in and then have smaller PRs after to build on top of it

Reply:
I can handle creating additional PR adding this script, after merging this PR 👍